external-crates/move/crates/move-compiler/tests/move_2024/naming/clever_errors_line_assert.move

```diff
@@ -0,0 +1,5 @@
+module 0x42::m { 
+    fun f() {
+        assert(false);
```

Comment:
Is this what you meant to test?
```suggestion
        assert!(false);
```

Reply:
I actually meant to test the non-macro `assert` call here -- this is checking that we don't insert the derived abort code.